src/App.tsx

```diff
@@ -13,6 +13,43 @@ import Community from './pages/Community';
 import * as css from './App.m.css';
 
 export default class App extends WidgetBase {
+	private _renderSectionOutlet(section: string) {
+		return [
+			<Outlet
+				key={`${section}`}
+				id={`${section}`}
+				renderer={(matchDetails) => {
+					if (matchDetails.isExact()) {
+						return <Section key={`section-${section}`} section={section} />;
+					}
+				}}
+			/>,
+			<Outlet
+				key={`${section}-page`}
+				id={`${section}-page`}
+				renderer={(matchDetails) => {
+					return (
+						<Section
+							key={`section-${section}`}
+							section={section}
+							path={`${section}/${matchDetails.params.page}`}
+						/>
+					);
+				}}
+			/>
+		];
+	}
+
+	/*private _renderPageOutlet(page: string) {
```

Comment:
did you mean to leave this here?

Reply:
Meant to have it so you can render a page without a "section". But did not have an example of that yet. To avoid the commented out code (to make ts happy) I will combine it with the other function.